packages/arb-token-bridge-ui/src/components/common/WithdrawalCountdown.tsx

```diff
@@ -25,8 +25,17 @@ export function WithdrawalCountdown({
   }
 
   if (nodeBlockDeadline === 'NODE_NOT_CREATED') {
-    const { isMainnet } = isNetwork(l1Network)
-    return <span>{isMainnet ? '~1 week' : '~1 day'} remaining</span>
+    const networkWithdrawalTimes: {
+      [chainId in ChainId]?: string
+    } = {
+      [ChainId.Mainnet]: '1 week',
+      [ChainId.Rinkeby]: '4 hours',
```

Comment:
Rinkeby withdrawal time is a bit more than a day.

You can always calculate it by multiplying `confirmPeriodBlocks` on an L2 network with the `blockTime` for the matching L1 network. You get the result in seconds and then you can convert it to any time measurement you need.